ppr-ui/src/components/unitNotes/UnitNotePanel.vue

```diff
@@ -15,9 +15,10 @@
                 :ripple="false"
               >
                 <span class="px-4">{{ isActive ? 'Hide Note' : 'View Note' }}</span>
-                <v-divider vertical class='mx-0' />
+                <v-divider v-if="noteOptions.length > 0" vertical class='mx-0' />
```

Comment:
It easier CSS wise to have the divider here rather than as part of the other button.